lib/DDG/Spice/MovieRuntime.pm

```diff
@@ -0,0 +1,17 @@
+package DDG::Spice::MovieRuntime;
+# ABSTRACT: Search for movie runtime
+
+use strict;
+use DDG::Spice;
+
+triggers startend => "movie runtime", "movie runnning time", "movie length";
+
+spice to => 'http://www.omdbapi.com/?t=$1&plot=short&r=json';
```

Comment:
TMDB, does it need any authorization to access the data??
On Aug 23, 2016 12:33 AM, "Zaahir Moolla" <notifications@github.com> wrote:

> In lib/DDG/Spice/MovieRuntime.pm
> <https://github.com/duckduckgo/zeroclickinfo-spice/pull/2968#discussion_r75738077>
> :
>
> > @@ -0,0 +1,17 @@
> > +package DDG::Spice::MovieRuntime;
> > +# ABSTRACT: Search for movie runtime
> > +
> > +use strict;
> > +use DDG::Spice;
> > +
> > +triggers startend => "movie runtime", "movie runnning time", "movie length";
> > +
> > +spice to => 'http://www.omdbapi.com/?t=$1&plot=short&r=json';
>
> I would suggest you use TMDB. We're using their API for a few other
> Instant Answers already 👍
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/duckduckgo/zeroclickinfo-spice/pull/2968/files/d2ee5fb179ae56b2b675c946a292b5ab96a1a008#r75738077>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AN6bqRxxSxyui-vgnyY2OBH2NfyjH4sJks5qifJpgaJpZM4JpKHu>
> .
>